### ISSUE TITLE
Auto focus of expand panel UI causes page jump

### ISSUE THREAD

**Issue by LlGC-jop** [UniversalViewer/universalviewer#1720]
**UV version:**

```
 universalviewer@4.x
```

**I'm submitting a:**

- [x] bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below
- [x] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**

When an embedded viewer loads the page 'jumps' down to it. This can be see here:

https://www.library.wales/discover-learn/education/learning-resources/thomas-picton-a-historical-enquiry

where the page will jump to one of the 3 viewers, somewhat randomly, after they've loaded.

The cause seems to be automatic focus of the left panel's open/close button. Because all viewers load at the same time, but with inconsistent finishes, which of the 3 buttons gets focus is random-ish.

**Expected behavior:**

Page should load and not jump.

**Steps to reproduce:**

https://www.library.wales/discover-learn/education/learning-resources/thomas-picton-a-historical-enquiry

or

Create a test page with either normal or embedded viewer, making sure that the page's content is long enough that the viewer is below the fold. Either left or right panel should be set to open by default e.g.

```
"moreInfoRightPanel": {
  "options": {
    "panelOpen": true
  }
},
"contentLeftPanel": {
  "options": {
    "panelOpen": true
  }
},
```

**Related code:**

```
// insert any relevant code here
```

**Other information:**

I think it's the `focus()` that happens after a panel is opened which causes this.

A simple fix would be to use the `allowStealFocus` config option to determine whether to do this or not, as it seems to match its intent. The code already includes a flag to say whether the toggle of the panel is automatic or not which would allow exclusion of non-auto focuses.

I'll add this and raise a PR for it, but as it's a potential accessibility issue we may want to have a community chat about it and decide whether a new config is needed so that existing configs aren't affected if UV is updated.

**Comment by demiankatz** [comment]
@LlGC-jop, it seems to me like this is a bug since it violates the documented intent of the existing `allowStealFocus` setting... so I'm not sure if a community discussion is strictly necessary before fixing it. Maybe we'd do just as well to go ahead with a fix, and then simply announce it to the community so they can make appropriate configuration decisions to match local needs.

**Comment by demiankatz** [comment]
Fixed by #1722.
